### PR TITLE
Misc update

### PR DESCRIPTION
**Pull request by fan-zhang-sv**

### CODE REVIEW

File: apps/testapp/src/context/CBWSDKProvider.tsx [hunk 1]
```diff
@@ -27,27 +28,41 @@ const dynamicallyImportSDK = (version: SDKVersionType) => {
...
+      const cbwprovider = cbwsdk.makeWeb3Provider('http');
+      setProvider(cbwprovider);
+    }
   }, [version]);
 
+  const setSDKVersion = (version: SDKVersionType) => {
+    localStorage.setItem(SELECTED_SDK_KEY, version);
+    setVersion(version);
+  };
+
```
Comment (fan-zhang-sv) [inline apps/testapp/src/context/CBWSDKProvider.tsx:59]: save selected sdkversion in localstorage, so sdk remians the same version on each refresh

File: packages/wallet-sdk/src/relay/walletlink/type/Web3Method.ts [hunk 2]
```diff
@@ -1,18 +1,21 @@
...
+  'scanQRCode',
+  'generic',
+  'childRequestEthereumAccounts',
+  'addEthereumChain',
+  'switchEthereumChain',
+  'makeEthereumJSONRPCRequest',
+  'watchAsset',
+  'selectProvider',
+  'connectAndSignIn',
+] as const;
```
Comment (fan-zhang-sv) [inline packages/wallet-sdk/src/relay/walletlink/type/Web3Method.ts:19]: we can derive union type from array, but not other way around. doing this so wallet can check is a string is valid web3method

File: packages/wallet-sdk/src/relay/walletlink/type/Web3Request.ts [hunk 3]
```diff
@@ -23,6 +23,9 @@ type _Web3Request =
         appLogoUrl: string | null;
       };
     }
+  | {
+      method: 'childRequestEthereumAccounts';
```
Comment (fan-zhang-sv) [inline packages/wallet-sdk/src/relay/walletlink/type/Web3Request.ts:27]: another type that is used between WL extension and RN, adding it here as the new `WalletLinkEventData` tightened the types for walletlinkConnection

File: packages/wallet-sdk/src/relay/walletlink/type/Web3Method.ts (hunk 2 above)
Comment (bangtoven) [reply packages/wallet-sdk/src/relay/walletlink/type/Web3Method.ts:19]: love it

**Review by bangtoven — APPROVED** [review]

**Review by bangtoven — APPROVED** [review]

**Review by bangtoven — APPROVED** [review]

**Pull request merged**